script.js

```diff
@@ -1,88 +1,73 @@
-let books = [
+let books = [];
+
+if (books === []) {
+  books = [
     {
-   titulo: '100 años de soledad',
-   author: 'Gabriel Garcia Marquez'
+      titulo: '100 años de soledad',
+      author: 'Gabriel Garcia Marquez',
     },
     {
-   titulo: 'El tirano',
-   author: 'Massimo Manfredy'
-    }
-]
-
-const bookList = document.querySelector('.books-display');
-let displ = ''
-books.forEach((el, index) => {    
-    displ += `
-    <div class="oneBook" id="book${index}">
-    <h4> ${el.titulo} </h4>
-    <h4> ${el.author} </h4> 
-    <button class="btn-display" id="${index}">remove</button> 
-    </div>
-    ` 
-
-});
-bookList.innerHTML = displ
-let removIds = [...document.querySelectorAll('.btn-display')].map(item => item.id)
-
-const newB = document.getElementById('bk')
+      titulo: 'El tirano',
+      author: 'Massimo Manfredi',
+    },
+  ];
+}
 
-newB.addEventListener('submit', function(event){
+function storage(ok) {
+  window.localStorage.setItem('localBooks', JSON.stringify(ok));
+}
 
-    event.preventDefault();
-    const reciTi = document.getElementById("recibe-ti").value
-    const reciAu = document.getElementById("recibe-au").value
+function getBooks() {
+  books = JSON.parse(window.localStorage.getItem('localBooks'));
+}
 
-    if(reciTi !== '' && reciAu !== ''){
-    let newBook = {
-        titulo: reciTi,
-        author: reciAu
-    }
-    books.unshift(newBook)
+getBooks();
 
-    let displ = ''
-    books.forEach((el, index) => {    
+function display() {
+  const bookList = document.querySelector('.books-display');
+  let displ = '';
+  books.forEach((el, index) => {
```

Comment:
- [ ] As previously suggested, this `display` method that is meant to display your books to the user is not working as expected. Please see error below and try to make it work in order to proceed 👍 
![image](https://user-images.githubusercontent.com/46839548/182447669-af71f741-5be0-4b52-8bb0-3ca00a6aab40.png)

Reply:
Ok, we understood now. There was a problem loading the original 'books' list, and with the rewritting of it with the localStorage changes. Now it should be fixed, thank for the observations